charts/spire/charts/spire-server/README.md

```diff
@@ -80,6 +84,7 @@ A Helm chart to install the SPIRE server.
 | nodeAttestor.k8sPsat.serviceAccountAllowList | list | `[]` |  |
 | nodeSelector | object | `{}` |  |
 | notifier.k8sbundle.namespace | string | `""` | Namespace to push the bundle into, if blank will default to SPIRE Server namespace |
+| persistence | object | `{"accessMode":"ReadWriteOnce","enabled":true,"size":"1Gi","storageClass":null}` | Enable persistence used by sqlite3 for spire-server, by the disk KeyStore, and/or by Tornjak. |
```

Comment:
Why did this get all condensed?

Reply:
Merge issue. Should be fixed.

Reply:
This gets condensed not because of merge issue, but because of the helm docs introduced.

See https://github.com/norwoodj/helm-docs#valuesyaml-metadata for more details.

In essence adding a documentation tag on any of the nested properties would resolve this again.

Reply:
No, there is that other persistence pr, and I manually merged part of it here to get them looking the same again, but put it in the wrong place. :/

I fixed the manual merge issue location and it should be back to split out again.